astropy/time/tests/test_mask.py

```diff
@@ -71,7 +71,7 @@ def test_str():
     t = Time(['2000:001', '2000:002'])
     t[1] = np.ma.masked
     assert str(t) == "['2000:001:00:00:00.000' --]"
-    assert repr(t) == "<Time object: scale='utc' format='yday' value=['2000:001:00:00:00.000' --]>"
+    assert repr(t) == "<Time object: scale='utc' format='yday' value=[2000:001:00:00:00.000 ... 0.0]>"
```

Comment:
This change does not look right to me - the masked elements should remain as masked, not `0.0`